matrix_client/api.py

```diff
@@ -345,7 +345,18 @@ def kick_user(self, room_id, user_id, reason=""):
         """
         self.set_membership(room_id, user_id, "leave", reason)
 
-    def set_membership(self, room_id, user_id, membership, reason=""):
+    def get_membership(self, room_id, user_id):
+        """Perform GET /rooms/$room_id/state/m.room.member/$user_id
+        Args:
+            room_id(str): The room ID
+            user_id(str): The user ID
+        """
+        return self._send(
+            "GET",
+            "/rooms/%s/state/m.room.member/%s" % (room_id, user_id)
+        )
+
+    def set_membership(self, room_id, user_id, membership, profile={}, reason=""):
```

Comment:
Its usually better to append new params, as then the function signature is fully backwards compat.